lib/codegen/Decision.cpp

```diff
@@ -466,12 +488,30 @@ llvm::Value *Decision::getTag(llvm::Value *val) {
   return res;
 }
 
+void DecisionNode::computeChoiceAncestors(std::vector<DecisionNode *> sorted) {
```

Comment:
I didn't write much C++ code in the last 4 years, but wouldn't this be better with a `const&` argument?